apps/meteor/app/api/server/v1/users.ts

```diff
@@ -555,6 +559,129 @@ API.v1.addRoute(
 	},
 );
 
+API.v1.addRoute(
+	'users.listByStatus',
+	{
+		authRequired: true,
+		validateParams: isUsersListStatusProps,
+		permissionsRequired: ['view-d-room', 'view-outside-room'],
+	},
+	{
+		async get() {
+			const { offset, count } = await getPaginationItems(this.queryParams);
+			const { sort, fields } = await this.parseJsonQuery();
```

Comment:
The `fields` params is deprecated for all endpoints, so I'm not sure if we should keep adding it to new endpoints :grimacing: 
If you don't plan to use it (which I think is the case), please remove it.